realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncedRealmTests.kt

```diff
@@ -538,6 +539,17 @@ class SyncedRealmTests {
         val expectedDecimal128List = RealmList<Decimal128>(Decimal128(10), Decimal128(100), Decimal128(20))
         val expectedObjectIdList = RealmList<ObjectId>(ObjectId(Date(1000)), ObjectId(Date(100)), ObjectId(Date(2000)))
         val expectedUUIDList = RealmList<UUID>(UUID.randomUUID())
+        val expectedMixedList = RealmList<Mixed>(
+                Mixed.valueOf(1.toLong()),
```

Comment:
Do we have full coverage here. Are mixed of data not possible? Anything actually checking that we test all the supported types?

Reply:
Missed the Date type.